content/en/docs/setup/install/multicluster/shared/index.md

```diff
@@ -203,8 +271,16 @@ spec:
         clusterName: ${REMOTE_CLUSTER_NAME}
       network: ${REMOTE_CLUSTER_NETWORK}
 
-      # Replace ISTIOD_REMOTE_EP with the the value of ISTIOD_REMOTE_EP set earlier.
+      # Replaces ISTIOD_REMOTE_EP with the the value of ISTIOD_REMOTE_EP set earlier.
       remotePilotAddress: ${ISTIOD_REMOTE_EP}
+
+  ## The ingressgateway is not required in the remote cluster if both clusters are on
```

Comment:
Is "ingressgateway" correct? In the next line of this comment, it says "ingress-gateway" with a hyphen; then in the commented-out lines, it says "ingressGateways" and "istio-ingressgateway". All four of those may be correct, I'm not sure. But because they're all different, I wanted to check.

Reply:
```suggestion
  ## The istio-ingressgateway is not required in the remote cluster if both clusters are on
```

Reply:
The service and deployment name is `istio-ingressgateway`. The config field is istioGateways.